### DISCUSSION TITLE
Help with my Giesen W6E

### DISCUSSION THREAD

**Discussion by MNCS2022** [artisan-roaster-scope/artisan#901] [General]
Hi everyone, 

Who could help me to conncet my IMac to to the roast machine, please? 

Best regards,

**Comment by MAKOMO** [comment]
Basic instructions are here: https://artisan-scope.org/machines/giesen/

Best to add a new network location on your Mac (System Preferences >> Network Control Panel) with an Ethernet entry set to "Manually". Assuming your roasters IP address is 172.30.30.10, put as IP address of the computer 172.30.30.42 and as network mask 255.255.255.0. Connect computer and roaster using an ethernet cable. Start Artisan, choose menu Config >> Giesen >> WxA, verify or correct the IP address of the roaster when asked by Artisan, click ON and check if you see the data in the LCDs. In case you have a Cropster enabled Giesen (an add-on to be payed to Giesen) you can use the WxA+ entry. In case your machine has an IR sensor, choose WxA-IR. In case your machine has an IR and Environmental temperature probe, choose WxA-IR-ENV.

Report!

**Reply by MAKOMO** [reply]
The roasters IP machine might as well be 172.30.30.50 or slightly different. Report.

**Comment by MNCS2022** [comment]
Hi Marko,

Many thanks for your quick answer.

How can I know if my Giesen has an IR sensor or an IR an
Environment temperature probe?

Best,






[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
18/06/22,
17:28:20

**Comment by aqdennis** [comment]
Hi,

An IR sensor is not standard on a W6E, you have to buy it separately. So if
you bought your W6E new from Giesen and didn't ask for it...you don't have
it.
Otherwise I guess you should be able to choose and monitor it from the
consol of the roaster. So if the consol doesn't show anything related to IR
readings it's safe to assume you don't have an IR sensor.

Cheers

**Comment by MNCS2022** [comment]
Hi Marko

It makes enough sense.

And what about the Environment probe? It's an internal probe to measure the
drum temperature, right?

Best,




[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
18/06/22,
17:43:43

**Reply by aqdennis** [reply]
I presume you're answering to me?
The ET probe is located at the front top. The BT probe is in the coffee outlet door on the front.

Cheers,
Dennis
(Also a W6E user)

**Comment by MNCS2022** [comment]
LoL.

Thanks for the info, mate.

Are you going to the WOC in Milan?

Nice to meet a W6E user.

Any tip that I have in mind Dennis.

Cheers,




[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
18/06/22,
18:13:06

**Reply by aqdennis** [reply]
No problem ☺️
We have our W6E since November 2020. It's a great roaster. We had a Toper before and we noticed that the Giesen blows out a lot of chaff through the exhaust pipe which the Toper didn't. So I don't know which one of the two is "normal"...but cleaning the pipes from the exhaust outside is something to regularly do. If not, you'll hear that the ventilator needs to go higher to maintain the underpressure in the drum. If the ventilator goes over 50% the whole machine starts seriously vibrating which can't be good. So keeping exhaust pipe and any external filters clean is the key...

We're not going to WOC Milan as we have other obligations. But usually we only go to the Amsterdam Coffee Festival as that is nearby. We can't afford being away too much time as so far 2022 is like coffeegeddon. Which after 2 not so good years with restaurant lockdowns is nice of course.

Cheers,
Dennis
Engrano Coffee Roasters
The Netherlands

**Comment by MNCS2022** [comment]
Ok. I'll take in count. I roast in a placer which doesn't have an
exhaust way. So I had to purchase an electrostatic filter for the hume. I'm
going to check the pipes once per month.

I'm starting and I usually roast 4 batches per week.  I usually use an
Aillio Bullet.

Well. If you have the opportunity of visit Cartagena  ( south eastern coast
of Spain) you can find me there roasting magic seeds in Mundo Novo Coffee
Shop.

Best,

Pedro Ros





[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
18/06/22,
19:49:26

**Comment by MNCS2022** [comment]
Hi mate, 

I've tried to connect my MacBook to Artisan and the programe show me this message 

Any though? 
![WhatsApp Image 2022-06-22 at 8 40 55 PM](https://user-images.githubusercontent.com/107749642/176467899-9a11a5ac-d424-43c6-8455-1770521ae1db.jpeg)

**Reply by poundy** [reply]
Hi,

That error means you can't talk to the roaster over TCP/IP. The most likely reason is you didn't follow the setup guide for the Giesen that Marko linked earlier, and didn't confirm the roaster IP is correct. Please tell us the steps you went through to confirm that things should work (ie what you verified was communicating, what settings you entered in the machine setup process etc) and how you confirmed the roaster's IP address and what your Mac's IP address is.

**Comment by MNCS2022** [comment]
Hi,

The IP of the Mac is 172.30.30.200,

I've tried configuring in WxA IR ENV and WxA IR plus ENV using an IP
172.30.30.30 and the I press the button START and the program show the
message of Recording but anything else.






[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Remitente
notificado con
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
30/06/22,
17:52:59

**Reply by poundy** [reply]
I know you get these updates via email, and your replies (sort of) make it into here, but if you could come into Github and reply in the proper box it would help keep the conversation better structured.

So you did no validation on what IP address is valid for your roaster? To me it seems obvious that you need to identify the correct IP to enter that in the software setup, and you haven't done so yet. There are a few ways that you can be certain: you can call Giesen and ask them what the correct default value is (and hope it wasn't overridden by someone, if this machine isn't new), or you could see if it can be set or changed on the control panel for the roaster, or you could use network analysis tools to identify the machine IP address (PING a bunch of different IPs on that range and look at the ARP table to see if any register), or you could just step through the setup in Artisan trying different options. Just possibly @aqdennis might have information that could shortcut this, if your machine is sufficiently similar to his and Giesen haven't changed a default, but I think it won't take too long to step through at least all the values that are listed for different modesl on the Artisan instruction page Marko sent earlier, they would be a great starting point in my view...

**Comment by MNCS2022** [comment]
Hello everyone, 

I've already install properly Artisan. 

I roasted a few of batch and I've noticed that the dry fase ( from charge to fellow) lasts around 4:50 and 5 and the Maillard reaction and the FC go very fast. 

@aqdennis. could you share with me a protocol to a full bact, please? 

My Charge temperature is 210C, Power 100%, RP80 and the Pa 120 and the first stage of roasting. Once I get the yellow I get down the power and rise the Pa. 

May should I have a higher CT? 

Thanks in advance

**Comment by aqdennis** [comment]
Hi,

Don't know how large your batch is, but 100% we really only use for 6kg
batches. For for instance a 1kg batch we won't go higher than 30%. But we
start scaling down the power shortly after TP, if you wait all the way to
DE it will go too fast, especially if you keep temperature at 210 degrees.
How fast your coffee heats up (hence roasts) depends on the temperature
difference ET-BT and with the power at 100% by the time you're at DE the ET
is already at your max of 210 degrees....from that point on it doesn't make
a difference anymore what you power setting is ... ET will remain at the
max temp even when power is set to 10%, and the only way to modify the
curve is bij changing the max temperature, not by power.
We usually load the batch at 190 degrees and keep the temperature to that
to get a more prolonged maillard and development (since ET is lower, ET-BT
is lower and thus coffee heats up slower). Only for a few coffees that we
know have a flattening curve in the end we increase the temperature a few
degrees at FC to avoid an RoR crash.

Be aware that increasing air flow speeds up maillard and development even
more ...so more reasons to have your max temperature lower than 210
degrees.

Hope that helps,
Dennis

**Reply by MNCS2022** [reply]
Hi Dennis, 

My weight of batch changes. I was talking about a protocol to roast 6kilos. In fact I use even a little more weight 6,30 totally to compensate the lost by the roasting process. 

Maybe I'm applying as a high charge temperature as a fast RP of the drum looking for a roast by convection. I used a Aillio Bullet and I still thinking that the W6E works in the same way. 

I'm going to lower the CT and the RP and keep the Pa and just change the power. 

My goal usually be to get the beginning of the yellow phase a round 3:30 to 4:00 min, starts the FC at 9:00 around 190C and finisnh the roast at 11:30 to 11:45 at 204C. 

Do you think that 80RP is too much fast? 

Best
Pedro

**Comment by aqdennis** [comment]
Hi Pedro,

If have the feeling you're roasting a lot darker than we do :) We roast
until we just before or about at second crack.
Anyway, you're doing several things to speed up the roasting process like
high air flow and high drum RPM combined with rather high charge
temperature so it's not really a surprise your roasting times are short.
We achieve DE at around 4:00 - 4:30 and FC at around 9:00, finishing
roasting at 11:30-12:00 at 192C using 65 RPM drum speed and 120 Pa under
pressure throughout the roast. Only for a few coffees that tend to get
smokey we increase the under pressure to 140 Pa when we reach FC (so not
already at DE to avoid the maillard phase going too fast).
If you want to roast until 204 degrees I would recommend when charging to
set the temperature to let's say 190 degrees and leave it like that until
at least ET reaches that temperature. After FC you could slowly increase
the temperature so that the ET will also slowly increase. After dropping at
the next charge you can set the machine temperature back to 190 degrees
again and repeat the same cycle. If you keep the machine at 210 the BT will
just go up too fast....

Good luck,
Dennis

**Comment by MNCS2022** [comment]
Hi Dennis, 

Thanks so much for your tips. I'll take in count and do them in the next session. 

I have other question maybe you can solve them. 

When do you load the beans into the drum? I mean how is your protocol between batchs? 

And how much time last the coffee to be enough cool in the cooling try? 

Kinds, 
Pedro

**Comment by aqdennis** [comment]
Hi Pedro,

Well, since we only roast until 190-195 degrees we can immediately charge
the next batch after dropping the current batch. But if we have to increase
the temperature after FC then after dropping we set the temperature back to
190C and keep the drop door open for 30s or so for the ET to drop before
charging again.
We leave a batch in the cooling tray until the next batch reaches DE. That
gives us just over 4 minutes to empty the cooling tray in a bucket and
store the bucket before the next batch reaches FC, from which point we want
to fully focus on the roasting again.

Cheers,
Dennis

**Reply by MNCS2022** [reply]
Hello Dennis, 

Thanks one more time. 

I'll tell you how is going with the new tips. 

Cheers, 
Pedro
